### PR TITLE
fix several bugs and improve documents

### PR DESCRIPTION
**Pull request by squirrelsc**
see commits for details.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -23,41 +24,69 @@ It depends on your contribution to LISA, you may need to learn more about LISA.
 
 ## Environment Setup
 
-Follow the [instal](install.md) steps to prepare the source code. Then follow the steps below to set up the corresponding development environment.
+Follow the [instal](install.md) steps to prepare the source code. Then follow
```
Comment (LiliDeng) [inline docs/development.md]: [install]

File: lisa/sut_orchestrator/azure/tools.py [hunk 2]
```diff
@@ -34,3 +34,24 @@ def get_version(self) -> str:
             result = self.run("-version")
         found_version = find_patterns_in_lines(result.stdout, [self.__version_pattern])
         return found_version[0][0] if found_version[0] else ""
+
+
+class VmGeneration(Tool):
+    """
+    This is a virtual tool to detect VM generation of Hyper-V techonology.
```
Comment (LiliDeng) [inline lisa/sut_orchestrator/azure/tools.py]: technology?

**Comment by glima** [comment]
Can't confirm/check anything about "fix lsvmbus failure", as the closest thing I'd have to reach that is WSL and I don't see the file (gen 1?).

**Comment by glima** [comment]
The rest should be fine/harmless.

**Comment by squirrelsc** [comment]
> Can't confirm/check anything about "fix lsvmbus failure", as the closest thing I'd have to reach that is WSL and I don't see the file (gen 1?).

Not sure about WSL. If the file doesn't exist, it means gen1 in Azure and Hyper-V.

**Pull request merged**